src/SpecFlow.NetCore/AppConfig.cs

```diff
@@ -18,59 +21,118 @@ internal class AppConfig
 		public const string SpecFlowSectionElement = "unitTestProvider";
 
 		public const string SpecFlowSection = @"	<specFlow>
-		<" + SpecFlowSectionElement + @" name=""xUnit"" />
+		<" + SpecFlowSectionElement + @" name=""{0}"" />
 	</specFlow>";
 
 		public static string Content = $@"<?xml version=""1.0"" encoding=""utf-8""?>
 <configuration>
 {SpecFlowSectionDefinition}
 {SpecFlowSection}
-</configuration>";
+</configuration>";
 		#endregion
-
+
 		public string Path { get; }
 
-		public AppConfig(string path)
+		public string TestFramework { get; }
+
+		public AppConfig(string path, string testFramework)
 		{
 			Path = path;
+			TestFramework = testFramework;
 		}
 
-		public static AppConfig CreateIn(DirectoryInfo directory)
+		public static AppConfig CreateIn(DirectoryInfo directory, out string usedTestRunner, string testRunner = null)
```

Comment:
I think we can get away without an `out` parameter, and just use the `TestFramework` property.